### DISCUSSION TITLE
Is there a way to detect when text is being entered in supabase realtime?

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#6339] [Questions] [answered]
Is there a way to detect when text is being entered in supabase realtime?

My ideal
https://i.gyazo.com/c46dffb3c70b2e89d4d7cf7d087d4491.gif

Reference.
https://multiplayer.dev/gHNYqqCqvY5ph4FNUzh62

**Comment by w3b6x9** [comment, marked as answer]
@ru88s I think you're looking for the Broadcast functionality of the new Realtime in order to achieve the same functionality as in the demo. There's no great way right now to do it. However, once we've enabled Broadcast for your project then you will be able to achieve the same effect.

**Reply by ru88s** [reply]
@w3b6x9 Thank you very much!

**Reply by w3b6x9** [reply]
@ru88s you're welcome! Broadcast is now available so feel free to check it out!
